Assets/Scripts/Properties/Properties.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public class Properties
```

Comment:
I recommend a comment similar to `Class that mimics the behavior of [Java properties](https://docs.oracle.com/en/java/javase/17/docs/api//java.base/java/util/Properties.html).`

Reply:
done